TTD/docs/TTDLiveRecorder.h/ILiveRecorder_AddCustomEvent.md

```diff
@@ -0,0 +1,71 @@
+# TTD::ILiveRecorder::AddCustomEvent method
```

Comment:
>r::AddCustomEvent method [](http://example.com/codeflow?start=19&length=24)

somewhere (here?) we should point to docs for accessing this information in the data model (link TBD)

Reply:
All across the board. Islands, the client itself, activities...

Reply:
(punt until we have that documentation)

Reply:
In case it is helpful, we have older docs on some TTD base objects such as Events - https://learn.microsoft.com/en-us/windows-hardware/drivers/debuggercmds/time-travel-debugging-event-objects

That topic includes a dx query example --> `dx -r2 @$curprocess.TTD.Events.Where(t => t.Type == "Exception").Select(e => e.Exception)`

Reply:
Thanx, but that is not related. Those events are internally populated by TTD to mark exceptions, thread creation and such.